src/content/platform-integration/ios/app-extensions.md

```diff
@@ -241,8 +243,26 @@ use an iOS simulator to test your extension in debug mode.
             view.addSubview(flutterViewController.view)
             flutterViewController.view.frame = view.bounds
         }
+
+        override func viewDidDisappear(_ animated: Bool) {
+            super.viewDidDisappear(animated)
```

Comment:
Some context on calling in `viewDidDisappear`: https://github.com/flutter/flutter/issues/165829#issuecomment-2787702265